src/state/reducers/index.ts

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux'
+import errorReducer from '../features/errorSlice'
+
+export const reducers = combineReducers({
```

Comment:
is `reducers` ever used? If not, please remove it

Reply:
hmm I don't know. But We can use it by editing store.ts.